app/security/Roles.scala

```diff
@@ -237,8 +237,8 @@ object RoleUtils {
 
   implicit def hc(implicit request: RequestHeader): HeaderCarrier = HeaderCarrier.fromHeadersAndSession(request.headers, Some(request.session))
 
-  def activeUserWithApp(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
-    user.user.isActive && user.application.isDefined
+  def activeUserWithActiveApp(user: CachedData)(implicit request: RequestHeader, lang: Lang) =
+    user.user.isActive && user.application.isDefined && user.application.fold(true)(_.applicationStatus != EXPORTED)
```

Comment:
you can use forAll if you want to have true when application is None ... am ok with fold(true) tooo :-)